packages/xrpc/src/client.ts

```diff
@@ -139,10 +140,11 @@ export async function defaultFetchHandler(
   try {
     // The duplex field is now required for streaming bodies, but not yet reflected
     // anywhere in docs or types. See whatwg/fetch#1438, nodejs/node#46221.
+    const headers = normalizeHeaders(httpHeaders)
     const reqInit: RequestInit & { duplex: string } = {
       method: httpMethod,
-      headers: httpHeaders,
-      body: encodeMethodCallBody(httpHeaders, httpReqBody),
+      headers,
+      body: encodeMethodCallBody(headers, httpReqBody),
```

Comment:
It looks like the implementation of `encodeMethodCallBody()` performs the normalization before checking the headers.  Is it necessary to perform the normalization here in `defaultFetchHandler()` too?